stix2/exceptions.py

```diff
@@ -49,3 +49,10 @@ class ImmutableError(STIXError, ValueError):
 
     def __init__(self):
         super(ImmutableError, self).__init__("Cannot modify properties after creation.")
+
+
+class VersioningError(STIXError, ValueError):
+    """Execption while using the Versioning API"""
+
+    def __init__(self, msg):
```

Comment:
You seem to be using "field", instead of "property" when naming things - is this intended or a slip?